### ISSUE TITLE
cucumber html meta data

### ISSUE THREAD

**Issue by a1dutch** [cucumber/cucumber#1317]
Hi,

How is it possible to pass the product name and product version meta data so its included in the new html report?

I'm using cucumber jvm 6.9.1 and the html plugin.

Cheers

Andy

**Comment by aslakhellesoy** [comment]
@a1dutch what's needed in order to close this issue?

**Comment by a1dutch** [comment]
@aslakhellesoy i think the following

* jenkins support #1253
* overriding of default meta data
* adding custom meta data

**Comment by aslakhellesoy** [comment]
@a1dutch how would you like to specify additional metadata? How/where would you like it rendered?

**Comment by a1dutch** [comment]
@aslakhellesoy maybe move the search/filter down onto its own line and span the whole width, you could then add them in their own box on the top right (where the filter is).  

Another option is to add them to the middle section (this might be pretty crowded, so it could be displayed in two columns and you could move the filter down as above).

you could pass these in as command line arguments

`-Dcucumber.plugin.react.info=App Version=1.2.3, App Commit=12da3fa2, Tested By=Andy`

or evironment variables

`CUCUMBER_PLUGIN_REACT_INFO=App Version=1.2.3, App Commit=12da3fa2, Tested By=Andy`

Cheers

Andy

**Comment by aslakhellesoy** [comment]
Ok, so if we use an environment variable like `CUCUMBER_PLUGIN_REACT_INFO=App Version=1.2.3, App Commit=12da3fa2, Tested By=Andy`, would you expect the value of that variable to be split on `,` and then rendered independently?

**Comment by a1dutch** [comment]
yes seperated by `,` you might want to support quoted strings like below (but i could live without that), and then rendered indepently.

`CUCUMBER_PLUGIN_REACT_INFO=App Version="1f23,1d53", App Commit='1f23,1d53', Tested By="Holland, Andy"`

i dont know if there is a standard way to pass parameters to a plugin, cucumber-jvm only seems to accept one parameter as part of the plugin and this is the report location

**Comment by davidjgoss** [comment]
cucumber-js supports `--format-options` as a JSON literal but I think it might be the only implementation that does.
